addons/mail/static/src/core/common/message_model.js

```diff
@@ -504,6 +502,16 @@ export class Message extends Record {
             message_id: message.id,
         });
     }
+
+    hideAllLinkPreview() {
+        rpc(
+            "/mail/link_preview/hide",
+            {
+                link_preview_message_ids: this.link_preview_message_ids.map((lp) => lp.id)
+            },
+            { silent: true }
+        );
+    }
```

Comment:
Nitpick: this hides all link previews, so plural (missing "s" at end of function name)